Sources/PointFreeTestSupport/PointFreeTestSupport.swift

```diff
@@ -492,6 +492,56 @@ extension Strategy {
   #endif
 }
 
+extension SnapshotTestCase {
+  public func assertSnapshots<A, B>(
+    of strategies: [String: Strategy<A, B>],
+    matching value: A,
+    record recording: Bool = false,
+    timeout: TimeInterval = 5,
+    file: StaticString = #file,
+    function: String = #function,
+    line: UInt = #line
+    ) {
+
+    strategies.forEach { name, strategy in
+      self.assertSnapshot(
+        of: strategy,
+        matching: value,
+        named: name,
+        record: recording,
+        timeout: timeout,
+        file: file,
+        function: function,
+        line: line
+      )
+    }
+  }
+
+  public func assertSnapshots<A, B>(
+    of strategies: [Strategy<A, B>],
+    matching value: A,
+    record recording: Bool = false,
+    timeout: TimeInterval = 5,
+    file: StaticString = #file,
+    function: String = #function,
+    line: UInt = #line
+    ) {
+
+    strategies.forEach { strategy in
+      self.assertSnapshot(
+        of: strategy,
+        matching: value,
+        named: name,
```

Comment:
does an index need to be appended to this or anything to prevent overwriting?

Reply:
Nah, named snaps don't have indices anymore.